play-frontend-hmrc-play-30/src/main/scala/uk/gov/hmrc/govukfrontend/views/viewmodels/servicenavigation/ServiceNavigation.scala

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.govukfrontend.views.viewmodels
+package servicenavigation
+
+import uk.gov.hmrc.govukfrontend.views.viewmodels.CommonJsonFormats._
+import uk.gov.hmrc.govukfrontend.views.viewmodels.JsonImplicits._
+import play.api.libs.functional.syntax._
+import play.api.libs.json._
+
+case class ServiceNavigation(
+  serviceName: Option[String] = None,
+  serviceUrl: Option[String] = None,
+  navigation: Seq[ServiceNavigationItem] = Seq(),
+  navigationClasses: String = "",
+  navigationId: String = "navigation",
+  navigationLabel: Option[String] = None,
+  classes: String = "",
+  attributes: Map[String, String] = Map.empty,
+  ariaLabel: String = "Service information",
```

Comment:
Where we have a default `ariaLabel` and `menuButtonText` in English, I think we should open a ticket to get these translated into Welsh as we have with other components. There's an example in `GovukPagination` here where we translate the aria properties: https://github.com/hmrc/play-frontend-hmrc/blob/main/play-frontend-hmrc-play-30/src/main/twirl/uk/gov/hmrc/govukfrontend/views/components/GovukPagination.scala.html#L28

We could look to do as part of this PR, or do as a future PR seeing as we're not expecting teams to start using this immediately in Production.